etc/config/analysis.defaults.properties

```diff
@@ -4,25 +4,28 @@ supportsBinary=false
 objdumper=
 # TODO: We might want to set a demangler, to deal with pasted compiler output with mangled symbols
 demangler=
+defaultCompiler=osacatrunkdef
 
-group.llvm-mca.compilers=llvm-mcatrunk
+group.llvm-mca.compilers=llvm-mcatrunkdef
```

Comment:
I nearly had a heart attack seeing an ID change...but it's in a defaults file 😓 so of course it's ok!